Core/src/org/sleuthkit/autopsy/datamodel/accounts/Accounts.java

```diff
@@ -153,7 +153,7 @@ private String getRejectedArtifactFilterClause() {
      *         based on the CasePreferences groupItemsInTreeByDataSource setting 
      */
     private String getFilterByDataSourceClause() {
-        if (Objects.equals(CasePreferences.getGroupItemsInTreeByDataSource(), true)) {
```

Comment:
change makes CasePreferences unused imports  (Objects is still used as an import in this class)